docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -401,11 +442,10 @@ def source(self, follow_additional_fields=True, document_type="pdf"):
                 new_content, done_with_content = self.field_list[0].get_single_field_screen(document_type)
                 content += new_content
             if self.field_list[0].field_type == 'end_attachment':
-                if hasattr(self, 'interview_label'):  # this tells us its the ending screen
-                  # content += "buttons:\n  - Exit: exit\n  - Restart: restart\n" # we don't want people to erase their session
-                  content += "need: " + self.interview_label + "\n"
-                  # TODO: insert the email code
-                  #content += "attachment code: " + self.attachment_variable_name + "['final']\n"
+                #if hasattr(self, 'interview_label'):  # this tells us its the ending screen
+                #  # content += "buttons:\n  - Exit: exit\n  - Restart: restart\n" # we don't want people to erase their session
+                #  # TODO: insert the email code
+                #  #content += "attachment code: " + self.attachment_variable_name + "['final']\n"
```

Comment:
We should just delete this whole section, right?

Reply:
I don't know enough about our future plans there. I think I'll punt that question up the line.